addons/test_website/tests/test_settings.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo
+import odoo.tests
+
+@odoo.tests.tagged('-at_install', 'post_install')
+class TestWebsiteSettings(odoo.tests.HttpCase):
+    def test_01_multi_website_settings(self):
+        self.env['website'].create([
+            {'name': "Website Test 1", 'specific_user_account': False},
```

Comment:
Pretty sure this one is useless and never actually part of your test, since we already have a default "My Website" in data which as `specific_user_account` set to `False` also.
Is will be this one selected by default when you enter website settings.
Probably just remove it and the test will run fine

Reply:
I thought that with `--without-demo=all` we didn't have a website and that maybe the runbot was executing the tests like that but in fact we still have a website even without demo data.
As you can see [here](https://github.com/odoo/odoo/compare/4d667f2cf53a72903d83bdd1bdd1b0072bd35aea..4b67df13f3f8e0789f0f92bcf7191f40bc9f3962), I removed it :+1:

Reply:
Yep, it's not a demo data but a real system data: you need to have at least one website to work, otherwise the system completely falls apart (and we have prevention of unlinking last website)